### DISCUSSION TITLE
ADE DeDRM not working in current install but working in old portable install

### DISCUSSION THREAD

**Discussion by bumpengrinder** [apprenticeharper/DeDRM_tools#1883] [Q&A]
Hi

I have a DRMd PDF in Adobe Digital Editions (4.5.11.187303) that will DeDRM fine with an old portable install of Calibre but not with my current, up to date, install.  The two setups are:

Calibre 5.29 + DeDRM 7.2.1
Calibre 4.83 portable + DeDRM 6.8.0

All on Windows 10.

Should it be working in the current setup?  If so, I'd please be grateful for tips to fix it.  Or shall I just carry on using the old setup to DeDRM things (and then move files across).

I did try reinstalling Calibre 5.29 to fix the issue but it didn't work.  However, I do not think I am getting a clean deinstall as when I reinstall it it already has my library (which is not in the default location) and the DeDRM plugins in place.

Many thanks.

**Comment by ElleKayEm** [comment]
DeDRM 7.2.1 still needs work for many Adobe DRM PDFs.  There is at least one pending pull request for this which others have reported worked for them.

**Reply by bumpengrinder** [reply]
Thank you, it is not just me then.
